src/odemis/driver/technolution.py

```diff
@@ -261,15 +566,10 @@ def __init__(self, name, role, parent, **kwargs):
         """
         super(MPPC, self).__init__(name, role, parent=parent, **kwargs)
 
-        self._server_URL = self.parent._server_url
-
         self._shape = (8, 8)
         self.filename = model.StringVA(time.strftime("default--%Y-%m-%d-%H-%M-%S"), setter=self._setFilename)
         self.dataContent = model.StringEnumerated('empty', DATA_CONTENT_TO_ASM.keys())
-        # NOTE: Do not write real username/password here since this is published on github in plaintext!
-        self.externalStorageURL = model.VigilantAttribute(urlparse('ftp://username:password@example.com/Pictures'),
-                                                          setter=self._setURL)
-        self.acqDelay = model.FloatContinuous(2.0, range=(0, 100000), unit='s', setter=self._setAcqDelay)
+        self.acqDelay = model.FloatContinuous(0.001, range=(0, 100000), unit='s', setter=self._setAcqDelay)
```

Comment:
Outdated, I left the same issue on a different commit open.
I think because the API does not provide a max this value was chosen to basically not have any limit present. However this changed.
Current implementation is:
self.scanDelay = model.TupleContinuous((0.0, 0.0), range=((0.0, 0.0), (200e-6, 10.0)), unit='s',
setter=self._setScanDelay)